### PR TITLE
Create 2023-04-24-t8code_version120_released.md

### PR DESCRIPTION
**Pull request by holke**

### CODE REVIEW

File: content/2023-04-24-t8code_version120_released.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
Title: t8code-1.2.0 released 
Date: 2023-04-24 17:00 
Category: News 
Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```

File: content/2023-04-24-t8code_version120_released.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+
+Title: t8code-1.2.0 released 
+Date: 2023-04-24 17:00 
+Category: News 
+Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```

File: content/2023-04-24-t8code_version120_released.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
+
+We have released version 1.3.0 of t8code!
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
We have released version 1.2.0 of t8code!
```

File: content/2023-04-24-t8code_version120_released.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
+
+We have released version 1.2.0 of t8code!
+
+New features include:
+
+...
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
- A new cmesh reader for VTK files.

- All of our test cases have been migrated to GoogleTest
  
- We updated to the newest p4est and sc versions

- Performance improvement for curved geometries
```

**Review by holke — COMMENTED** [review]
It is not working yet, the news does not show on the homepage.

File: content/2023-04-24-t8code_version120_released.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
+
+We have released version 1.2.0 of t8code!
+
+New features include:
+
+- A new cmesh reader for VTK files.
```
Comment (jmark) [inline content/2023-04-24-t8code_version120_released.md:10]: Only this bullet point ends with a period. Decide which style to use for all bullet points.

**Review by jmark — CHANGES_REQUESTED** [review]

File: content/2023-04-24-t8code_version120_released.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+- All of our test cases have been migrated to GoogleTest
+  
+- We updated to the newest p4est and sc versions
+
+- Performance improvement for curved geometries
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
- All of our test cases have been migrated to GoogleTest.
  
- We updated to the newest p4est and sc versions.

- Performance improvement for curved geometries.
```

File: content/2023-04-24-t8code_version120_released.md (hunk 5 above)
Comment (holke) [reply content/2023-04-24-t8code_version120_released.md:10]: done

File: content/2023-04-24-t8code_version120_released.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
Title: t8code-1.2.0 released 

Date: 2023-04-24 17:00 

Category: News 

Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```

File: content/2023-04-24-t8code_version120_released.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+
+
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
```

File: content/2023-04-24-t8code_version120_released.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
+Title: t8code-1.2.0 released 
+
+Date: 2023-04-24 17:00 
+
+Category: News 
+
+Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
Title: t8code-1.2.0 released 
Date: 2023-04-25 14:00 
Category: News 
Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
```

File: content/2023-04-24-t8code_version120_released.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+- A new cmesh reader for VTK files.
+
+- All of our test cases have been migrated to GoogleTest.
+  
+- We updated to the newest p4est and sc versions.
+
+- Performance improvement for curved geometries.
+
+For the full change log see: https://github.com/DLR-AMR/t8code/releases/tag/v1.2.0
```
Comment (holke) [inline content/2023-04-24-t8code_version120_released.md]:
```suggestion
Click <a href="https://github.com/DLR-AMR/t8code/releases/tag/v1.2.0">here</a> for the full change log.
```

**Review by jmark — APPROVED** [review]

**Pull request merged**